pytest.ini

```
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --ignore-glob="*colorama*"
```

Would it not make more sense just to add the `tests` folder as `testpaths`? I reckon that should limit the tests run to that folder (thus ignoreing `colorama`). Although, the addopts solution should also work).
```suggestion
testpaths = tests
```

---

Okay, just looked at the documentation and it does seem to say it will be exclusive, which I had not realized. Let's try it.